test/builder-dir-sharding.js

```diff
@@ -91,7 +91,7 @@ module.exports = (repo) => {
             expect(nodes[0].path).to.be.eql(expectedHash)
             expect(mh.toB58String(nodes[0].hash)).to.be.eql(expectedHash)
             expect(nodes[1].path).to.be.eql(expectedHash + '/b')
-            expect(nodes[1].size).to.be.eql(21)
+            expect(nodes[1].size).to.be.eql(29)
```

Comment:
Why does this need to change if the test is the same?

Reply:
Had to change this for `ls` because go-ipfs reports the size as reported by the link, not the what we get from the `file.size` accessor... Still have to investigate which one is the correct one. Any clues on why these values differ?

Reply:
I see, I believe that was added for ipfs.add and it seems it also made it to .ls. LGTM then :)

Reply:
👍